libs/ui/src/modal.test.tsx

```diff
@@ -115,10 +115,12 @@ describe('Modal', () => {
 
   it('handles after open', () => {
     // Work around a react-modal bug: https://github.com/reactjs/react-modal/issues/903
-    jest.spyOn(window, 'requestAnimationFrame').mockImplementation((cb) => {
```

Comment:
TS v5.5 flags `cb` as having type `any`, presumably because `window.requestAnimationFrame` is overloaded?